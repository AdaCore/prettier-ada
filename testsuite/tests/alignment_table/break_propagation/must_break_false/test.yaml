title: Table_Alignment Must_Break propagation
driver: document_formatter
description: |
  Test that Hardlines inside Aligment_Tables propagate to the whole table.
