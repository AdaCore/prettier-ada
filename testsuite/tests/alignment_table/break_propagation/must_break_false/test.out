A
B C
D
A
BBC
D
